lib/letter_avatar/configuration.rb

```diff
@@ -21,9 +21,17 @@ def colors_palette
     end
 
     def colors_palette=(v)
-      @colors_palette = v if v.in?(Colors::PALETTES)
+      @colors_palette = v if Colors::PALETTES.include?(v)
     end
 
+		def custom_palette
+			@custom_palette ||= nil
+		end
+
+		def custom_palette=(v)
+			@custom_palette = v
```

Comment:
IMHO, `custom_palette` validation should be done here. thanks to this you'll get an exception on initialization, not when using the gem in view ;)